pkgs/development/tools/casperjs/default.nix

```diff
@@ -28,9 +28,9 @@ in stdenv.mkDerivation rec {
     make test
   '';
 
-
   installPhase = ''
-    mv $PWD $out
+    mkdir $out
```

Comment:
I hope this clears things up a bit:

                                    | old `mv`        | new `mv`        | `cp`
----------------------------------- | --------------- | --------------- | ---------------
same mount point (e.g. laptop)      | somewhat faster | somewhat faster | somewhat slower
different mount point (e.g. server) | doesn't work    | same            | same

From the admittedly naive table I see that the use of `mv` here is the best choice. At first I thought one reason not to do it is to avoid inventing new ways of doing things, but that doesn't seem to be relevant after a little grepping.

Now, you said it doesn't matter that much, and I agree. Just say that you prefer `cp` and I'll make the change.